apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -263,9 +256,11 @@ export default {
     },
   },
   created() {
-    if (this.defaultValue) {
-      this.internalValues = deepClone(this.defaultValue);
-    }
+    this.tableMetaData.columns.forEach((column: IColumn) => {
```

Comment:
this seems okay to move it here.

Reply:
This is done to trigger setting the default value only once, the old location triggered on every key press (setting back the original default value internally and not reflecting the reset in the user interface ).